hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py

```diff
@@ -50,10 +47,9 @@ def _get_overlapping_sheet_domain(self):
     def _check_overlapping_sheets_project_id(self):
         self._check_overlapping_sheets()
 
-    @api.multi
     @api.constrains("company_id", "project_id")
     def _check_company_id_project_id(self):
-        for rec in self.sudo():
+        for rec in self.with_user(SUPERUSER_ID):
```

Comment:
@pedrobaeza see, not only I treat that instruction like that :)

Reply:
@davejames `sudo()` is not deprecated, `sudo(other_that_superuser)` is

Reply:
Please propose an alternate redaction for that.

Reply:
Easy!

> Replace `sudo(user)`, keep `sudo()`: "deprecated use of sudo(user), use with_user(user) instead"

Reply:
thanks good to know

Reply:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0/_compare/ac45f661a0aad55f05ed5b72f9e24d654022e18b...e548c9f190738f52d6ef0c5954556fba38205033